docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -0,0 +1,256 @@
+.. _module_dev_tutorial_sample
+
+Ansible Module Windows Development Walkthrough
+==============================================
+
+In this section, we will walk through developing, testing, and debugging an
+Ansible Windows module.
+
+Because Windows modules are written in Powershell and needs to be run on a
+Windows host, this guide differs from the usual development walkthrough guide.
+
+What's covered in this section:
+
+-  `Environment setup <#environment-setup>`__
+-  `New module development <#new-module-development>`__
+-  `Playbook module testing <#playbook-module-testing>`__
+-  `Debugging (remote) <#debugging-remote>`__
+-  `Unit testing <#unit-testing>`__
+-  `Integration testing <#integration-testing>`__
+-  `Communication and development support
+   <#communication-and-development-support>`__
+
+
+Environment setup
+=================
+
+TODO: Add in more information on how to use Vagrant to setup a Windows host.
+
+
+New module development
+======================
+
+When creating a new module there are a few things to keep in mind of
```

Comment:
'of' probably not needed - 'When creating a new module there are a few things to keep in mind'